### PR TITLE
Development

### PR DESCRIPTION
**Pull request by kshitij-ingale**

### CODE REVIEW

File: evaluate_images.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+path = sys.argv[1]
+abs_path = os.path.abspath(path)+'/'
+file_names = os.listdir(path)
+file_loc = [abs_path + x for x in file_names]
+for file in file_loc:
+	img = cv2.imread(file)
+	print(img.shape)
+
+	# fasdas
```
Comment (JMHOO) [inline evaluate_images.py]: this file looks like useless :)

File: src/config.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
```
Comment (JMHOO) [inline src/config.py:2]: - Class level vars should all capitalized. 
```
class image_properties(object):
    DEPTH = 3
    HEIGHT = 512
    WIDTH = 512
    COMPRESSED_DIMS = [1, 32, 32, 8]
```

Comment (JMHOO) [reply src/config.py:2]: `config_train` and `config_test` look like shared lots of configs, you can use a little bit inherit or mixin to simplify the code.

File: src/data.py [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    def load_dataframe(filename, load_semantic_maps=False):
+        df = pd.read_hdf(filename, key='df').sample(frac=1).reset_index(drop=True)
+
+        # if load_semantic_maps:
+        #     return df['path'].values, df['semantic_map_path'].values
+        # else:
+        #     return df['path'].values
+        return df['path'].values
+
+    @staticmethod
```
Comment (JMHOO) [inline src/data.py:15]: I'm wondering would this be an error here?

**Comment by JMHOO** [comment]
I literally don't know which files are your contribution, it would be better to mark where the files original from.

**Comment by JMHOO** [comment]
The README also need to be updated, and add some least unit tests as well

File: evaluate_images.py (hunk 1 above)
Comment (kshitij-ingale) [reply evaluate_images.py]: Yeah, it was a temp. I guess I missed it while pushing the repo

File: src/config.py (hunk 2 above)
Comment (kshitij-ingale) [reply src/config.py:2]: Yes, I think I would modify it so that there is just one config

**Pull request merged**